exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/ExtendedOptionIterator.java

```diff
@@ -52,14 +55,44 @@
  *  only the value set at SESSION level.
  */
 public class ExtendedOptionIterator implements Iterator<Object> {
-//  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(OptionIterator.class);
+  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(ExtendedOptionIterator.class);
+  private static final String OPTIONS_DESCRIBE_PROPERTIES = "options.describe.properties";
+  private static final Properties descriptionMap = new Properties();
 
   private final OptionManager fragmentOptions;
   private final Iterator<OptionValue> mergedOptions;
+  private HashMap<OptionValue.Kind,String> typeMapping;
+  private Map<OptionScope, Integer> preference;
+
+  //Load & Provide a descriptions map
+  public static Properties getDescriptionMap() {
+      if (descriptionMap.isEmpty()) { //Expect a one time load
+        synchronized (descriptionMap) {
+          try (InputStream optionDescInputStream = OptionIterator.class.getClassLoader().getResourceAsStream(OPTIONS_DESCRIBE_PROPERTIES)) {
+            descriptionMap.load(optionDescInputStream);
+          } catch (IOException e) {
+            logger.warn("Unable to load descriptions from {}", OPTIONS_DESCRIBE_PROPERTIES);
```

Comment:
Add exception to logger.

Reply:
My bad. I think I missed adding it in the refactor.